src/features/cseMachine/animationComponents/FrameCreationAnimation.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { Group } from 'react-konva';
+
+import { GenericArrow } from '../components/arrows/GenericArrow';
+import { ControlItemComponent } from '../components/ControlItemComponent';
+import { Frame } from '../components/Frame';
+import { StashItemComponent } from '../components/StashItemComponent';
+import { Text } from '../components/Text';
+import { ArrayValue } from '../components/values/ArrayValue';
+import { PrimitiveValue } from '../components/values/PrimitiveValue';
+import { Value } from '../components/values/Value';
+import { CseAnimation } from '../CseMachineAnimation';
+import { Config } from '../CseMachineConfig';
+import { ControlStashConfig } from '../CseMachineControlStashConfig';
+import { defaultActiveColor, defaultStrokeColor, isEnvEqual } from '../CseMachineUtils';
+import { Animatable, AnimationConfig } from './base/Animatable';
+import { AnimatedGenericArrow } from './base/AnimatedGenericArrow';
+import { AnimatedRectComponent, AnimatedTextComponent } from './base/AnimationComponents';
+import { getNodeDimensions, getNodeLocation, getNodePosition } from './base/AnimationUtils';
+
+export class FrameCreationAnimation extends Animatable {
```

Comment:
Can we add a `/** ... */` comment here to help explain the animation for consistency with the other animation classes, but more importantly, for clarity?

Reply:
Added a short description here, and same for the function application animation.